src/phoenix/server/main.py

```diff
@@ -250,7 +251,7 @@ def _get_pid_file() -> Path:
         initial_spans=fixture_spans,
         initial_evaluations=fixture_evals,
     )
-    server = Server(config=Config(app, host=host, port=port))
+    server = Server(config=Config(app, host=host, port=port))  # type: ignore
```

Comment:
filed an issue @anticorrelator - this seems strange to force a none type in here: https://github.com/Arize-ai/phoenix/issues/3062

Reply:
Here's the context behind this solution that Dustin had found: https://github.com/encode/uvicorn/discussions/1529#discussioncomment-4848949

Reply:
@anticorrelator could we add a comment linking this issue?